bigframes/ml/sql.py

```diff
@@ -304,6 +304,11 @@ def ml_predict(self, source_sql: str) -> str:
         return f"""SELECT * FROM ML.PREDICT(MODEL {self._model_ref_sql()},
   ({source_sql}))"""
 
+    def ml_explain_predict(self, source_sql: str) -> str:
```

Comment:
add a unit test to make sure we don't accidentally break it.

https://github.com/googleapis/python-bigquery-dataframes/blob/main/tests/unit/ml/test_sql.py

Reply:
https://github.com/googleapis/python-bigquery-dataframes/blob/33928008f64ddc4ae88432bcff6daa7ec64e807c/tests/unit/ml/test_sql.py#L333-L343 for an example.